lib/spoom/sorbet/metrics.rb

```diff
@@ -58,7 +58,7 @@ def files_count
 
       sig { params(key: String).returns(T.nilable(Integer)) }
       def [](key)
-        metrics[key]
+        metrics[key] || 0
```

Comment:
Let's display clean `0` instead of an empty strings

Reply:
There is a neater trick for this: Initialize the hash for `metrics` with `Hash.new(0)` instead of as `{}`.